src/code/z_debug.c

```diff
@@ -1,32 +1,32 @@
 #include "global.h"
 
 typedef struct {
-    u8 x;
-    u8 y;
-    u8 colorIndex;
-    char text[21];
-} DbCameraTextBufferEntry; // size = 0x18
+    /* 0x0 */ u8 x;
+    /* 0x1 */ u8 y;
+    /* 0x2 */ u8 colorIndex;
+    /* 0x3 */ char text[21];
+} DebugCameraTextBufferEntry; // size = 0x18
 
 typedef struct {
-    u16 hold;
-    u16 press;
+    /* 0x0 */ u16 hold;
+    /* 0x2 */ u16 press;
 } InputCombo; // size = 0x4
 
 RegEditor* gRegEditor;
 
-DbCameraTextBufferEntry sDbCameraTextBuffer[22];
+DebugCameraTextBufferEntry sDebugCamTextBuffer[22];
 
-s16 sDbCameraTextEntryCount = 0;
+s16 sDebugCamTextEntryCount = 0;
 
-Color_RGBA8 sDbCameraTextColors[] = {
-    { 255, 255, 32, 192 },  // DBCAMERA_TEXT_YELLOW
-    { 255, 150, 128, 192 }, // DBCAMERA_TEXT_PEACH
-    { 128, 96, 0, 64 },     // DBCAMERA_TEXT_BROWN
-    { 192, 128, 16, 128 },  // DBCAMERA_TEXT_ORANGE
-    { 255, 192, 32, 128 },  // DBCAMERA_TEXT_GOLD
-    { 230, 230, 220, 64 },  // DBCAMERA_TEXT_WHITE
-    { 128, 150, 255, 128 }, // DBCAMERA_TEXT_BLUE
-    { 128, 255, 32, 128 },  // DBCAMERA_TEXT_GREEN
+Color_RGBA8 sDebugCamTextColors[] = {
+    { 255, 255, 32, 192 },  // DBG_CAM_TEXT_YELLOW
```

Comment:
For the sake of consistency I think `sDebugCamXxx` can be `sDebugCameraXxx`, it doesn't matter too much anyway

I would say the same for `DBG_CAM_TEXT_` -> `DEBUG_CAMERA_TEXT_`, even if it matters less (but if that's really worth shortening we should consider `DbgCamera_` instead of `DebugCamera_`, that would be a lot more impactful, so... `DEBUG_CAMERA_TEXT_` ?)

Reply:
fwiw I think "debug cam" rolls of the tongue much nicer, and is what I call it normally. Wouldnt be against calling it that in general everywhere.
But, do understand why that wouldnt be desirable

Reply:
fwiw I wouldn't be against DebugCam everywhere

this is not a critical system so I don't have a critical opinion :sunglasses: (don't call it "EyeAtGoWeee" though)

Reply:
I think my preference would be to still have the `DebugCamera_` full name in function names, but I'd be okay with `debugCam` for everything else, even typedef struct names. (although if others want the `DebugCam_` prefix, that's alright too).

Reply:
`DebugCamera_` for functions and `debugCam`/`DebugCam`/`DEBUG_CAM` everywhere else works for me :+1:

Reply:
Works for me 👍